eng/CodeCoverage.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <EnableCodeCoverage>true</EnableCodeCoverage>
```

Comment:
I think there should be a condition otherwise this is always true and overriding project config.

Reply:
I think that we disable bulk for all project if we disable or?

Reply:
Fine for me to disable globally only.